test_scripts/API/Restructuring_OnResetTimeout/016_BC_OnResetTimeout_with_wrong_requestID_and_HMI_did_not_respond.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 13000) with right methodName but wrong requestID to SDL
+-- 4) HMI does not send response in 10 seconds after receiving request
+-- SDL does:
+-- 1) Respond in 11 seconds with GENERIC_ERROR resultCode to mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Restructuring_OnResetTimeout/common_OnResetTimeout')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local wrongRequestID = "Wrong_RequestID"
+
+local paramsForRespFunction = {
+  notificationTime = 6000,
```

Comment:
@HSavynetska for right time calculating _notificationTime_ should be = 0

Reply:
@LitvinenkoIra Update